### PR TITLE
[fix](cloud) Fix hdfs accessor build error

### PR DESCRIPTION
**Pull request by platoneko**
## Proposed changes

- Fix hdfs accessor build error
- Remove CHECK in refresh storage vault info

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/cloud/cloud_storage_engine.cpp [hunk 1]
```diff
@@ -296,7 +296,36 @@ void CloudStorageEngine::sync_storage_vault() {
 void CloudStorageEngine::_refresh_storage_vault_info_thread_callback() {
     while (!_stop_background_threads_latch.wait_for(
             std::chrono::seconds(config::refresh_s3_info_interval_s))) {
+<<<<<<< HEAD
         sync_storage_vault();
+=======
```
Comment (gavinchou) [inline be/src/cloud/cloud_storage_engine.cpp:301]: conflict block !

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by platoneko** [comment]
run buildall

File: be/src/cloud/cloud_storage_engine.cpp [hunk 2]
```diff
@@ -275,7 +275,11 @@ void CloudStorageEngine::sync_storage_vault() {
         return;
     }
 
-    CHECK(!vault_infos.empty()) << "no s3 infos";
+    if (vault_infos.empty()) {
+        LOG(WARNING) << "no storage vault info" << st;
```
Comment (gavinchou) [inline be/src/cloud/cloud_storage_engine.cpp:279]: missing space before `st`

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**